client/blocks/login/index.jsx

```diff
@@ -387,26 +369,26 @@ class Login extends Component {
 					);
 				} else {
 					headerText = <h3>{ translate( "Let's get started" ) }</h3>;
+					const poweredByWpCom =
+						wccomFrom === 'nux'
+							? translate( 'All Woo Express stores are powered by WordPress.com!' )
+							: translate( 'All Woo stores are powered by WordPress.com!' );
```

Comment:
I'd suggest to temporarily update this to `Log in to WooCommerce.com with your WordPress.com account to connect your store and manage your extensions` until we confirm a better copy to fix the misleading messaging.

Reply:
I agree, that wording seems better in this situation. I think this is what the UI used to say.

<img width="500" src="https://github.com/Automattic/wp-calypso/assets/1647564/b1b2d144-e519-4002-b679-df29b9634018">

Reply:
A bit of a pickle here because the new layout shows the account selector if wpcom account is logged in, and there's the sign up link for the user if they don't have an existing wpcom account. 

It would look like:

<img width="763" alt="image" src="https://github.com/Automattic/wp-calypso/assets/27843274/973b00c0-408f-4289-bace-7872cb152179">
<img width="694" alt="image" src="https://github.com/Automattic/wp-calypso/assets/27843274/090b9408-d447-437a-a2eb-7e576c29e584">

which is a bit repetitive

Reply:
I don't suppose we can do like

```
Log in to WooCommerce.com with your WordPress.com account to connect your store and manage your extensions.
Don't have an account? Sign up.
```

? If it's not easy to change the second line, we can go with `All Woo stores are powered by WordPress.com!` for now, thanks.

Reply:
I find the "All Woo stores are powered" text a bit weird – WooCommerce is still the name of the application, and you don't need to be connected to WPCOM to run a WooCommerce store. I guess in future we can add a new `wccom-from` param and have a text like `Woo.com is powered by WordPress.com` that is better suited to WCCOM.

Reply:
The writing experts are now suggesting this for the text. This may not fit neatly with the existing logic – or maybe it will help simplify things.

### Woo Express signup and log in

> All Woo Express stores are powered by WordPress.com. 
> Please log in to continue. Don’t have an account? [Sign up](link).

### Everything else

> To connect to services, manage extensions, and more, please log in using your WordPress.com account. Don’t have an account? [Create one](link).

Reply:
I had a bit of an issue with a dangling word due to the width constraint so I took a bit of liberty with the wording, hope this is ok?

### Non-NUX flows:
###### Sign up
![image](https://github.com/Automattic/wp-calypso/assets/27843274/82365c9a-e15c-49ee-ac67-63183a30e575)

###### Log in
![image](https://github.com/Automattic/wp-calypso/assets/27843274/b8548156-3cda-4d1a-9cb1-f21854475a2c)


### NUX
###### Sign up
![image](https://github.com/Automattic/wp-calypso/assets/27843274/85206f68-26e7-4f89-ad32-d910085c991b)

###### Log in
![image](https://github.com/Automattic/wp-calypso/assets/27843274/594a601e-79f7-44ab-a55f-59e144076433)

Reply:
^ pinging @andfinally in case it got lost in your notifications 😬